scripts/main.js

```diff
@@ -23,8 +23,13 @@ app.whenReady().then(() => {
     if (BrowserWindow.getAllWindows().length === 0) createWindow();
   })
 
-  // Prevent ESC from exiting fullscreen
-  globalShortcut.register('Esc', () => { });
+  globalShortcut.register("Shift+`", () => {
+    var fs = BrowserWindow.getAllWindows()[0].isFullScreen();
+    if(fs)
+      BrowserWindow.getAllWindows()[0].setFullScreen(false);
+    else
+      BrowserWindow.getAllWindows()[0].setFullScreen(true);
+  });
```

Comment:
could be done in one line `BrowserWindow.getAllWindows()[0].setFullScreen(!BrowserWindow.getAllWindows()[0].isFullScreen());`
i'm not good with js :)

Reply:
Personally I have a C#/.NET background where having long single-line code or method calls within method calls is usually frowned upon.

A small "refactoring" will be done later to conform with the JavaScript standards but there are some other parts of the (previous) code that need changing and I'd prefer addressing them all in one PR.

So for now I've merged the branch as it was.